flink-tests/src/test/java/org/apache/flink/test/checkpointing/SavepointITCase.java

```diff
@@ -153,6 +153,47 @@ public void setUp() throws Exception {
         }
     }
 
+    @Test
+    public void testStopSavepointWithFlip27Source() throws Exception {
```

Comment:
As I mentioned https://github.com/apache/flink/pull/15425#discussion_r603854862 we need also a test for stopping with savepoint with draining.